### DISCUSSION TITLE
How to set the zip file comment?

### DISCUSSION THREAD

**Discussion by nijinekoyo** [gildas-lormeau/zip.js#503] [Q&A] [answered]
I did not find the `comment` configuration in the `ZipWriterConstructorOptions` type. Currently I can only set the `comment` of each file in `zipWriter.add()`. I hope to set a single zip in `new ZipWriter()` `comment` of the file

**Comment by bwbroersma** [comment, marked as answer]
See [ZipWriter close](https://gildas-lormeau.github.io/zip.js/api/classes/ZipWriter.html#close), the optional first argument is the comment for the EOCD:
https://github.com/gildas-lormeau/zip.js/blob/7021dd3a036853fabd3dc733a30b7b823e87d5d4/lib/core/zip-writer.js#L201-L203
https://github.com/gildas-lormeau/zip.js/blob/7021dd3a036853fabd3dc733a30b7b823e87d5d4/lib/core/zip-writer.js#L163

**Comment by nijinekoyo** [comment]
> See [ZipWriter close](https://gildas-lormeau.github.io/zip.js/api/classes/ZipWriter.html#close), the optional first argument is the comment for the EOCD:
> 
> https://github.com/gildas-lormeau/zip.js/blob/7021dd3a036853fabd3dc733a30b7b823e87d5d4/lib/core/zip-writer.js#L201-L203
> 
> 
> https://github.com/gildas-lormeau/zip.js/blob/7021dd3a036853fabd3dc733a30b7b823e87d5d4/lib/core/zip-writer.js#L163

That's it, thanks
